src/riakc_ts_get_operator.erl

```diff
@@ -46,6 +46,8 @@ deserialize({error, {Code, Message}}) when is_integer(Code), is_atom(Message) ->
     {error, {Code, iolist_to_binary(atom_to_list(Message))}};
 deserialize({error, Message}) ->
     {error, Message};
+deserialize(tsgetresp) ->
+    {ok, {[], []}};
```

Comment:
Indeed. Sorry I didn't see your PR in time (it had taken me a good hour to arrive at the same conclusion).